src/App.js

```diff
@@ -1,18 +1,40 @@
-import React from "react";
-import "./App.css";
-import BarraBusca from "./componentes/BarraBusca/BarraBusca";
-import ContainerCategorias from "./componentes/ContainerNotas/ContainerNotas";
-import FormCadastro from "./componentes/FormCadastro/FormCadastro";
-function App() {
-  return (
-    <div className="App">
-      <BarraBusca />
-      <section class="conteudo_container">
-        <FormCadastro />
-        <ContainerCategorias />
-      </section>
-    </div>
-  );
+import React, { Component } from "react";
+import "./assets/App.css";
+import BarraBusca from "./componentes/BarraBusca";
+import ContainerNotas from "./componentes/ContainerNotas";
+import FormCadastro from "./componentes/FormCadastro";
+import { Notas } from "./dados/Notas";
+class App extends Component {
+  constructor() {
+    super();
+    this.notas = new Notas();
+    this.state = { notas: [] };
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <BarraBusca />
+        <section className="conteudo_principal">
+          <FormCadastro criarCard={this.criarCard.bind(this)} />
+          <ContainerNotas
+            dados={this.state.notas}
+            apagarCard={this.apagarCard.bind(this)}
+          />
+        </section>
+      </div>
+    );
+  }
+
+  criarCard(titulo, texto) {
```

Comment:
Normalmente esses métodos são deixados pra cima do render. Isso acontece por conta do padrão de leitura inicial mesmo. Então a ideia é deixar o que está sendo utilizado mais pra cima. No React isso não faz tanta diferença, mas se for fazer isso aqui com js puro fora de classe faz.